pkg/liveshare/session.go

```diff
@@ -23,14 +22,14 @@ type LiveshareSession interface {
 	KeepAlive(string)
 	OpenStreamingChannel(context.Context, ChannelID) (ssh.Channel, error)
 	StartSharing(context.Context, string, int) (ChannelID, error)
+	GetKeepAliveReason() chan string
```

Comment:
Any idea why this is a `chan`?

Reply:
That's just what's used for the `keepAliveReason`, but there's not really a need for it to be a `chan` so I went ahead and updated it to just return a string.